src/node.cc

```diff
@@ -1383,27 +1384,28 @@ void AppendExceptionLine(Environment* env,
                      linenum,
                      sourceline_string);
   CHECK_GE(off, 0);
```

Comment:
If the total length of all of the strings being printed to `arrow` exceeds `sizeof(arrow)`, `snprintf()` will return `-1`. When that happens (like in #2581), `CHECK_GE(off, 0)` will trigger and cause a C++ assertion because.